tests/platform_tests/fwutil/fwutil_common.py

```diff
@@ -81,14 +83,23 @@ def complete_install(duthost, localhost, boot_type, res, pdu_ctrl, auto_reboot=F
             logger.info("Rebooting switch using {} boot".format(boot_type))
             duthost.command("sonic-installer set-default {}".format(current))
             reboot(duthost, pdu_ctrl, boot_type, pdu_delay)
+            logger.info("Waiting on switch to shutdown...")
+            localhost.wait_for(host=hn, port=22, state='stopped', delay=1, timeout=60)
```

Comment:
Can we leave the timeout=timeout which the user passed as argument ? Here now I see you changed to 60 sec - instead can let the caller decide ?

Otherwise changes LGTM

Reply:
Hi @judyjoseph, the timeout user passed as argument is for the async run of the fwutil command. The command should finish in this timeout(1200s by default). But the timeout=60 in line#87 is only for the ssh down in the power off reboot, for which 60s should be enough.

Reply:
Hi @judyjoseph, since the default timeout is 1200s, if somehow the power off reboot doesn't work(it depends on the PDU in the lab), we will waste a lot of time waiting here. What do you think if I increase this timeout to 120s?